<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Submission</class>
 <widget class="QDialog" name="Submission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_lastName"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_userName"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_password"/>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBox_female">
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_male">
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spinBox_age"/>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton_submit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Submission&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
